test/featuregates_linter/cmd/feature_gates.go

```diff
@@ -167,7 +167,11 @@ func verifyOrUpdateFeatureList(rootPath, featureListFile string, update, version
 	}
 
 	if diff := cmp.Diff(featureList, baseFeatureList); diff != "" {
-		return fmt.Errorf("detected diff in unversioned feature list, diff: \n%s", diff)
+		if versioned {
+			return fmt.Errorf("detected diff in versioned feature list, diff: \n%s", diff)
+		} else {
+			return fmt.Errorf("detected diff in unversioned feature list, diff: \n%s", diff)
```

Comment:
nit: Might make sense to add the file path as well in the error message, ex:
```
return fmt.Errorf("detected diff in unversioned feature list (%s), diff: \n%s", unversionedFeatureListFile, diff)
```

Reply:
Hi Aaron, I'll surely add that :)